cmd/minikube/cmd/status.go

```diff
@@ -35,12 +36,21 @@ var statusCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		api := libmachine.NewClient(constants.Minipath, constants.MakeMiniPath("certs"))
 		defer api.Close()
-		s, err := cluster.GetHostStatus(api)
+		ms, err := cluster.GetHostStatus(api)
 		if err != nil {
 			glog.Errorln("Error getting machine status:", err)
 			os.Exit(1)
 		}
-		fmt.Fprintln(os.Stdout, s)
+		ls := "N/A"
+		if ms == state.Running.String() {
+			ls, err = cluster.GetLocalkubeStatus(api)
+		}
+		if err != nil {
+			glog.Errorln("Error getting machine status:", err)
+			os.Exit(1)
+		}
+		fmt.Fprintln(os.Stdout, "minikubeVM: "+ms)
```

Comment:
Can you explain with an example what the output looks like with or without the flag or what parameters format takes?  Are you suggesting that format hides parts of the outputs or is it something like json, etc.?